Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+Synopsys/Imperas OVPSim can be considered for other configurations.
+
+Gateways between OpenHW core-v-verif repositories and Thales internal environment (GitLab, CI...) will be set up.
+
+The verification sub-projects plans will provide more insight into the verification methods.
+
+## Future enhancements (off project):
+
+Increasing the core architectural performance (out-of-order...), adding RISC-V extensions, adhering to a RISC-V profile, delivering a larger subsystem (SMP multi-core, peripherals, accelerators...)
+
+## Explanation of why OpenHW should do this project
+
+Like RI5CY, the ARIANE core donated by ETH Zürich and U. Bologna was at the heart of the OpenHW Group creation. We need an application processor core in our portfolio.
+
+Also, there are very few FPGA technology-independent softcores and CV32A6 is an alternative to proprietary cores.
+
+## Industry landscape: description of competing, alternative, or related efforts in the industry
+
+On the core side, for ASIC targets, these are the most comparable competitors (pipeline length, MMU, single issue...):
+- ARM: Cortex-A5
+- SiFive: U54
+- CHIPS Alliance&nbsp;/ Western Digital: CVA6 between the smaller SweRV  EL2 and the larger EH1.
+- ANDES: A25 (32-bit) and AX25 (64-bit)
+- Frontgrade Gaisler: NOEL-V
+
+On the FPGA side, CV32A6 is a technology-independent alternative to these proprietary cores:
+- Xilinx: Microblaze
+- MicroChip: Mi-V
+- Intel: Nios-II, Nios-V
+
+On the tool side, CHIPS Alliance has plans to make their tools open-source
+([link](https://semiengineering.com/components-for-open-source-verification/), 5<sup>th</sup> paragraph).
+
+### Market differentiators
+
+To differentiate from the competition, marketing can stress:
+- Source code written in **SystemVerilog**, a widely accepted language;
+- **Open-source** availability of the cores;
+- Open-source availability of verification artefacts, which is a great step towards certification for security- and safety-critical applications;
+- The availability of a family of **technology-independent cores optimized for ASIC and FPGA targets**;
+- The ability to **extend the instruction** set thanks to the CV-X-IF interface;
+- The **SW ecosystem** running on CVA6, with FreeRTOS and Linux already demonstrated;
+- The ability to scale to SMP multi/many-core CPUs thanks to the OpenPiton framework;
+- The permissive licence scheme that allows the integration in open-source or closed-source projects or the addition of a "secret sauce";
+- The low exposition to export control
+([OpenHW Group Membership Agreement](https://www.openhwgroup.org/membership/openhw-group-membership-agreement-2019-10-16.pdf), section 4.1).
+
+## External dependencies
+
+The project relies on:
+- Related and sister OpenHW projects (joint with other cores): LLVM, FreeRTOS, core-v-verif, CV-X-IF specification, OpenPiton...
+- Open-source software: GCC, GDB, LLVM, Linux (Yocto, BuildRoot), OpenSBI, UBoot, BBL...
```

Comment:
CV-MESH is the OpenHW version of OpenPiton

Reply:
I'll reword OpenPiton/CV-MESH as OpenPiton is well known.